closed/OpenJ9.gmk

```diff
@@ -333,11 +336,13 @@ run-preprocessors-j9 : stage-j9 \
 build-j9 : run-preprocessors-j9
 	@$(ECHO) Compiling OpenJ9 in $(OUTPUT_ROOT)/vm
 ifeq ($(OPENJ9_ENABLE_CMAKE),true)
-	(export OMR_DIR=$(OUTPUT_ROOT)/vm/omr OPENJ9_BUILD=true && \
+	(export OPENJ9_BUILD=true $(EXPORT_MSVS_ENV_VARS) && \
 		cd $(OUTPUT_ROOT)/vm/build && $(MAKE) $(MAKEFLAGS) install \
 	)
 else
-	(export OPENJ9_BUILD=true $(EXPORT_NO_USE_MINGW) && cd $(OUTPUT_ROOT)/vm && $(MAKE) $(MAKEFLAGS) all)
+	(export OPENJ9_BUILD=true $(EXPORT_NO_USE_MINGW) $(EXPORT_MSVS_ENV_VARS) && \
```

Comment:
Please split these lines consistently with other command sequences (`&&` at the start of a line with the next command).

Reply:
Fixed!